base/prometheus/prometheus.ConfigMap.yaml

```diff
@@ -299,19 +312,23 @@ data:
           - alert: GitserverDiskSpaceLowCritical
             expr: src_gitserver_disk_space_available / src_gitserver_disk_space_total < 0.05
             labels:
-              severity: page
+              level: critical
```

Comment:
Can you clarify what this does for me? I thought that `severity: page` made these page OpsGenie via our current deployment, so I think this change would mean that these alerts (which we currently rely on to keep sourcegraph.com healthy) would stop paging us via OpsGenie

Reply:
I haven't applied this change in https://github.com/sourcegraph/deploy-sourcegraph-dot-com/pull/2839 yet with this in mind, but my thinking is long-term, these alerts will stay and be treated as just another `critical` alert, with all `critical` alerts going to our OpsGenie

Reply:
There is an expectation that the contents of this repository are always the authoritative source and the dot-com repository is merely additions / edits to that; in the spirit of that I would say that changing this should wait until it can be done in the dot-com repository as well

(I like the idea and think it is good otherwise)

Reply:
makes sense! f5398fdf adds the `severity: page` back